packages/twenty-front/src/modules/ui/layout/show-page/components/ShowPageRightContainer.tsx

```diff
@@ -94,6 +96,9 @@ export const ShowPageRightContainer = ({
     CoreObjectNameSingular.Company,
     CoreObjectNameSingular.Person,
   ].includes(targetObjectNameSingular);
+  const isWorkflow =
```

Comment:
I feel the logic of this component is wrong. You should not have to pick like this, it should rely on backend. 
I would:
- add the fields in the targetableObject (I think you have them in the parent)
- loop on the fields instead. If you have the field noteTarget, you display notes. If you have the field taskTarget, you display tasks...
- we need to remove `eventTargets` in the workflow entity since we do not use them and we do not want them displayed with this logic

Wdyt @martmull?

Reply:
Yes it would be much cleaner, might be the good moment to improve that component logic